Bookish/Models/EditBooks.cs

```diff
@@ -2,49 +2,41 @@
 
 public class EditBooks
 {
-    public static void InsertBooks(string author, string title)
+    static LibraryContext context = new LibraryContext();
+    public static List<BookModel> GetBookList()
     {
-        using var context = new LibraryContext();
-        var book = new BookModel
-        {
-            Author = author,
-            Title = title
-        };
-        context.Books.Add(book);
+        var books = context.Books.ToList();
+        return books;
+    }
+    
+    public static BookModel GetBookFromId(int id)
+    {
+        BookModel book = context.Books.Single(b =>b.Id == id);

```

Comment:
What if a book with this ID doesn't exist?

Reply:
Good point - I've added error catching to the list of things to deal with